e2e/internal/client/cli.go

```diff
@@ -185,8 +211,9 @@ func (cli *CLI) Init(opts *InitRequest) (string, error) {
 			"--type", opts.WorkloadType,
 			"--tag", opts.ImageTag,
 			"--dockerfile", opts.Dockerfile,
-			"--port", opts.SvcPort,
-			deployOption))
+			deployOption,
+			scheduleOption, opts.Schedule,
+			portOption, opts.SvcPort))
```

Comment:
oh so if these guys are not specified then all these params become the empty string right? and I guess that doesn't affect the arguments to the cmd